### ISSUE TITLE
Learning Rate Decay Method

### ISSUE THREAD

**Issue by kuan-wang** [ericsun99/MobileNet-V2-Pytorch#1]
Hello, That's really a good result :)  Can you share your training method to train mobilenetv2 that high?( such as default learning rate and weight-decay method) . Thanks~
I am currently training the model with SGD and keeping other hyper-parameters the same (except that I use batch size 256), but only get Top1: 71.17

**Comment by ericsun99** [comment]
I have training with SGD, and have using some data augment method.

**Comment by SophieZhou** [comment]
@ericsun99 which data augmentation did you use during your training? Using crop, flip, resize, I only got about top 1 accuracy 68 % and top5 about 88%. 
And I evaluate the model by center crop.

**Comment by SophieZhou** [comment]
@ericsun99 @THUKey In your test, do you test the model with tencrops?

**Comment by Coderx7** [comment]
@THUKey ,@ericsun99 could you please share your hyperparameters?

**Comment by CF2220160244** [comment]
Hello  @THUKey . I use SGD train with a 1080Ti, batchsize=96, lr=0.045, weight decay=0.00004, and decrease the lr 0.98 for each epoch, after 2 days only get 67%, can you tell me your GPU, batchsize, and the initial lr to help me.
I am a student in beijing institute of technology. Thank you very much!
